README.md

```diff
@@ -50,7 +50,7 @@ For a general picture about the server configuration, including the project setu
 
 ### Path to server executable
 
-If your server is manually installed and not on your path, you can also manually set the path to the executable.
+If your server is manually installed and not on your path, you can also [manually](https://code.visualstudio.com/docs/getstarted/settings#_settings-editor) set the path to the executable (not to the `...-wrapper`!) in [settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson).
```

Comment:
Why not the wrapper?

Reply:
IDK why setting the path to a wrapper triggered a message similar to one from this issue "https://github.com/haskell/haskell-language-server/issues/1342". When I set to just hls binary and reloaded the window, the message disappeared.

Reply:
That doesn't seem right.